client/tests/component/specs/testWorkspaceUserRole.spec.ts

```diff
@@ -0,0 +1,37 @@
+// Parsec Cloud (https://parsec.cloud) Copyright (c) BUSL-1.1 2016-present Scille SAS
+
+import { it } from 'vitest';
+import { mockI18n, mockRouter, getDefaultProvideConfig } from '@tests/component/support/mocks';
+
```

Comment:
Add same comment for consistency
```suggestion

// Before importing anything else
```

Reply:
still missing